pubspec.yaml

```diff
@@ -1,15 +1,19 @@
 name: http
-version: 0.12.2
+version: 0.12.3-nullsafety
```

Comment:
I think we are generally going with breaking changes for non-allow-listed packages, although that isn't totally decided

Reply:
Oh it sounds like from the changelog entry this is in the allow list, even so we really only don't want to do breaking changes for flutter depended on packages.

Reply:
Flutter doesn't depend on this directly – at least I don't think so.

A lot of stuff in the Flutter ecosystem does, though...

Reply:
cc @natebosch I would still do a major version bump.

We need to stop setting a bad example here.

Reply:
> Oh it sounds like from the changelog entry this is in the allow list

It isn't.

> I would still do a major version bump.

+1. We'll want a breaking change for https://github.com/dart-lang/http/issues/424 in any case. I can try to get to that next week so we can make a decision before we need to publish this.